src/utils/mod.rs

```diff
@@ -287,13 +287,20 @@ pub fn parse_emoji(mention: impl AsRef<str>) -> Option<EmojiIdentifier> {
         return None;
     }
 
-    if mention.starts_with("<:") && mention.ends_with('>') {
+    if (mention.starts_with("<:") || mention.starts_with("<a:")) && mention.ends_with('>') {
         let mut name = String::default();
         let mut id = String::default();
+        let mut animated = false;
 
-        for (i, x) in mention[2..].chars().enumerate() {
+        if &mention[1..3] == "a:" {
+            animated = true;
+        }
+
+        let start = animated as usize + 2;
```

Comment:
This is clever, but I prefer readability more.
```suggestion
        let start = if animated { 3 } else { 2 }; 
```